### PR TITLE
Adding Piston and Compsoc articles for 17/6/18

### PR DESCRIPTION
**Pull request by gurupunskill**

### CODE REVIEW

File: blog_src/_posts/2018-06-19-string-genetic.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+## Fitness
+
+The fitness of an individual is proportional to how close it is to the defined string S for this environment.  
+  
+
+```
+	Individual = "ABCDEFGHIJKLMNOPQRSTUVWXYS"  
+	Fitness(Individual) = (Number of characters in Individual same as String)  
+			      / (Number of characters in String)
```
Comment (salman-bhai) [inline blog_src/_posts/2018-06-19-string-genetic.md]: I think maybe you could reformat this once properly?

File: blog_src/_posts/2018-06-19-string-genetic.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            i+=1
+        return "".join(child)
+```
+  
+  
+### Wrapping it all up
+  
+After concretely defining our environment, to genrate a random population and choosing the best indiviuals and breeding them to create the next generation is trivial.  
+
+Check out the full notebook [here](https://github.com/gurupunskill/string-matcher/)
```
Comment (salman-bhai) [inline blog_src/_posts/2018-06-19-string-genetic.md]: Maybe have it as a gist?

**Comment by archit-p** [comment]
Why are there so many merge conflicts? Looks like there was some issue with the build. Can't be merged directly. Should we post the articles ourselves?

**Comment by salman-bhai** [comment]
The merge conflicts are primarily due to the earlier builds. Will have to resolve locally and then merge.

**Pull request merged**